packages/react-composites/src/composites/CallWithChatComposite/adapter/CallWithChatAdapter.ts

```diff
@@ -582,8 +588,19 @@ export interface CallWithChatAdapterSubscriptions {
   off(event: 'chatParticipantsAdded', listener: ParticipantsAddedListener): void;
   off(event: 'chatParticipantsRemoved', listener: ParticipantsRemovedListener): void;
   off(event: 'chatError', listener: (e: AdapterError) => void): void;
+
+  // CallWithChat subscriptions
+  on(event: 'chatInitialized', listener: ChatInitializedListener): void;
+  off(event: 'chatInitialized', listener: ChatInitializedListener): void;
 }
 
+/**
+ * Callback for {@link CallWithChatAdapterSubscribers} 'chatInitialized' event.
+ *
+ * @public
+ */
+export type ChatInitializedListener = (event: { adapter: CallWithChatAdapter }) => void;
```

Comment:
Nit: can we unify these names with the state or the event? (either reference in the name here `adapter` or dont)